autogen/retrieve_utils.py

```diff
@@ -40,7 +40,23 @@
     "yml",
     "pdf",
 ]
-UNSTRUCTURED_FORMATS = ["docx", "doc", "odt", "pptx", "ppt", "xlsx", "eml", "msg", "epub"]
+UNSTRUCTURED_FORMATS = [
```

Comment:
Why is "csv", "md", and "tsv" unstructured? I suggest move CSV, TSV, and MD to structured.

With the `partition` function call and `"\n".join` below, the library only replace comma with newline character, which is even more unstructured. 

Similarly, the structured architecture (e.g., sections, lists) in markdown files are also removed with the package.

Reply:
Thanks. Removed csv, md and tsv from UNSTRUCTURED_FORMATS.